lib/hal/video.c

```diff
@@ -198,7 +198,13 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 	
 	DWORD dwAdapter = dwEnc & 0x000000FF;
 	DWORD dwStandard = dwEnc & 0x0000FF00;
-	
+
+        bool is_480p;
+        bool allow_480p = dwEnc & VIDEO_MODE_480P;
+        bool allow_480i = !allow_480p;
```

Comment:
If you also use `!is_480p` below (instead of `!is_480i`), then why not also use `!allow_480p`(instead of `allow_480i`)?

This is also different from the 480p, 720p and 1080i modes. So a comment explaining why another check for 480p is necessary, would be good (`If 480p is enabled, reject 480i modes (forces a 480p mode to be picked)`).